### PR TITLE
feat: add dreyerx mainnet registry

### PR DESCRIPTION
**Pull request by billalxcode**
### Description
Add DreyerX Mainnet Chain

### Testing

Manual

### CODE REVIEW

File: chains/dreyerx/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiKey: abc
+    apiUrl: https://scan.dreyerx.com/api
+    family: blockscout
+    name: DreyerX Scan
+    url: https://scan.dreyerx.com
```
Comment (paulbalaji) [inline chains/dreyerx/metadata.yaml:7]: api key not necessary
```suggestion
  - apiUrl: https://scan.dreyerx.com/api
    family: blockscout
    name: DreyerX Scan
    url: https://scan.dreyerx.com
```

File: chains/dreyerx/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+domainId: 23451
+isTestnet: false
+name: dreyerx
+nativeToken:
+  decimals: 18
+  name: DreyerX
+  symbol: DRX
+deployer:
+  name: billalxcode
+  url: https://github.com/billalxcode
```
Comment (paulbalaji) [inline chains/dreyerx/metadata.yaml:19]: this will fail linter, please run `yarn lint:fix` and `yarn prettier`

File: chains/dreyerx/logo.svg
Comment (paulbalaji) [inline chains/dreyerx/logo.svg:1]: this file looks very large, please consider reducing the size by running through an svg minifer https://jakearchibald.github.io/svgomg/

**Comment by billalxcode** [comment]
@paulbalaji I've fixed everything, please review again

File: chains/dreyerx/logo.svg
Comment (billalxcode) [reply chains/dreyerx/logo.svg:1]: I've replaced it with a new one and I've also changed the size to 64px

File: chains/dreyerx/metadata.yaml (hunk 2 above)
Comment (billalxcode) [reply chains/dreyerx/metadata.yaml]: i've fixed

File: chains/dreyerx/metadata.yaml (hunk 1 above)
Comment (billalxcode) [reply chains/dreyerx/metadata.yaml]: I've deleted apiKey

**Comment by billalxcode** [comment]
can you approve my pull request? @paulbalaji

**Pull request closed without merging**